tests/NetEscapades.EnumGenerators.NetStandard/NetEscapades.EnumGenerators.NetStandard.csproj

```diff
@@ -18,5 +18,6 @@
 
   <ItemGroup>
     <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
+    <PackageReference Include="System.Memory" Version="4.6.3" />
```

Comment:
tl;dr; this is the breaking change that I don't want to introduce 😄

Reply:
What if it's only for the targets that benefit from that? Chances are users are already using that.

Reply:
You can keep it as is for now and later introduce static extension methods to support implicit conversions to and from `string` and `ReadOnlySpan<char>` when targeting the .NET 10 SDK. That said, we should bear in mind that .NET 8 will remain supported for nearly 18 more months.

Reply:
> You can keep it as is for now and later introduce static extension methods to support implicit conversions to and from `string` and `ReadOnlySpan<char>` when targeting the .NET 10 SDK. That said, we should bear in mind that .NET 8 will remain supported for nearly 18 more months.

I really don't follow? 😅 Definitely don't want implicit conversions from enums to string, that's not what you mean, right? And yeah, if you don't have the .NET 10 SDK then you just don't get the extension member versions. Not sure what that has to do with this PR though? 😅

Reply:
Not from and to an `enum` to `string`. From and to `string` and `ReadOnlySpan<char>` for .NET Standard 2.0. And that would be a file scoped extension.